crates/optimism/rpc/Cargo.toml

```diff
@@ -15,7 +15,8 @@ workspace = true
 # reth
 reth-evm.workspace = true
 reth-primitives-traits.workspace = true
-reth-provider.workspace = true
+reth-storage-api.workspace = true
+reth-chain-state.workspace = true
```

Comment:
added `reth-chain-state` crate due to reasoning below

Reply:
great